jenkins/release_ci/opensearch_maven_release/maven-sign-release.jenkinsfile

```diff
@@ -0,0 +1,94 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211130'
+            alwaysPull true
+        }
+    }
+    parameters {
+        string(
```

Comment:
Let's make this is a choice rather than a string input, since this would always be a fixed value of the dropdown list

Reply:
I would prefer we leave what it is right now. As @peternied said, we don't know what architectures we could potentially support in the future and it takes efforts to track and update choices of all jobs.

Reply:
Okay. I suggested a menu since it creates confusion as to what values needs to be filled in, which in this case is only `x64`. 

Out of curiosity, is it possible to have a centralized parameterized list for cases like these, params for a job?

Reply:
Upon further looking into this - if don't provide a dropdown list, the user can enter any value for `ARCH` and `platform`. So, one can save and create artifacts of any platform and architecture given there is no check for the values input as a string. This can be both deceiving and dangerous.

For example, I can simple enter `arm64` and the job would run fine and save artifacts as `arm64` which is incorrect.

Reply:
In that case why give an option to enter the ARCH? If we know arm64 won't work or something we do not want, `x64` is the hardcode value that can go here.

Reply:
Is the signer currently only working on `x64` platform? I'm not sure why other architecture is incorrect.

Reply:
We upload only x64 on maven. arm64 is out of the picture. I do not know the reason.

Reply:
Exactly, but this can potentially have `arm64` as input, which is incorrect